contracts/Comptroller/Comptroller.sol

```diff
@@ -1423,18 +1373,14 @@ contract Comptroller is ComptrollerV9Storage, ComptrollerInterfaceG2, Comptrolle
             && user != 0x24e77E5b74B30b026E9996e4bc3329c881e24968,
             "Blacklisted"
         );
-
+        
```

Comment:
```suggestion

```

Reply:
We could leave this for now. I'm going to add solidity linting as soon as this and 185 and 140 are merged